app.js

```diff
@@ -1,42 +1,52 @@
-const express = require("express")
-const {getHealthResponse, getApiInformation} = require("./controllers/api-controller")
-const {getTopics} = require("./controllers/topics-controller")
-const {getArticle, getArticles} = require("./controllers/articles-controller")
+const express = require("express");
+const {
+  getHealthResponse,
+  getApiInformation,
+} = require("./controllers/api-controller");
+const { getTopics } = require("./controllers/topics-controller");
+const {
+  getArticle,
+  getArticles,
+} = require("./controllers/articles-controller");
+const { getComments, postComment } = require("./controllers/comments-controller");
 
+const app = express();
 
-const app = express()
+app.use(express.json())
 
-app.get("/api/healthcheck", getHealthResponse)
+app.get("/api/healthcheck", getHealthResponse);
 
-app.get("/api/topics", getTopics)
+app.get("/api/topics", getTopics);
 
-app.get("/api", getApiInformation)
+app.get("/api", getApiInformation);
 
-app.get("/api/articles", getArticles)
+app.get("/api/articles", getArticles);
 
-app.get("/api/articles/:article_id", getArticle)
+app.get("/api/articles/:article_id", getArticle);
 
+app.get("/api/articles/:article_id/comments", getComments);
 
+app.post("/api/articles/:article_id/comments", postComment);
 
 
 // errors
 
 app.use((err, req, res, next) => {
-    if(err.status && err.message){
-        res.status(err.status).send({message: err.message})
-    }
-    next(err)
-})
-
-app.use((err, req, res, next)=> {
-    if(err.code === "22P02"){
-    res.status(400).send({message: "Bad request"})
-    }
-    next(err)
-})
+  if (err.status && err.message) {
+    res.status(err.status).send({ message: err.message });
+  }
+  next(err);
+});
 
 app.use((err, req, res, next) => {
-    res.status(500).send("internal server error")
-})
+  if (err.code === "22P02") {
+    res.status(400).send({ message: "Bad request" });
+  }
+  next(err);
+});
 
-module.exports = app
\ No newline at end of file
+app.use((err, req, res, next) => {
+  res.status(500).send("internal server error");
+});
+
```

Comment:
^^ I think you should move your error handling middleware to a new file now, to prevent app.js from becoming too busy with erorr handlers! :)